src/partials/40_section-sale.html

```diff
@@ -0,0 +1,46 @@
+<div class="section--sale">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-6">
+        <div class="image-box large">
+          <div class="image" style="background-image: url(images/sale/sale-image1.jpg)">
+            <div class="layer">
+              <div class="layer-content">
+                <h2>Guest room</h2>
+                <h1>Sofa</h1>
+                <div class="discount">-20%</div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-6">
+        <div class="image-box small-top">
+          <div
+            class="image"
```

Comment:
To nie jest uwaga, ale zastanawiam się, czy nie lepiej byłoby podzielić klasy .image na 3 osobne, ponieważ występują one w 3 różnych konfiguracja i mają różne style. Chodzi mi o hipotetyczną sytuację, kiedy zmieniając/dodając atrybut dla klasy globalnej posypią się 2 pozostałe elementy. Nie jestem pewny, czy moje rozumowanie jest dobre, więc wołam @OskarLebuda albo @tvardy, żeby ew. wyprowadzili mnie z błędu :D

Reply:
Ta prośba o poprawę dotyczy wcięć w pliku 40_section-sale.html

Reply:
Ok, racja z tymi klasami. Zrobiłam porządek :)